userspace/libsinsp/test/sinsp.ut.cpp

```diff
@@ -511,6 +513,23 @@ TEST_F(sinsp_with_test_input, execveat_invalid_path)
 	}
 }
 
+// Check that the path in case of execve is correctly overwritten in case it was not possible to collect it from the
+// entry event but it is possible to collect it from the exit event
+TEST_F(sinsp_with_test_input, execve_invalid_path_entry)
```

Comment:
I think this can be the right time to split this test file into multiple pieces; wdyt?